pubspec.yaml

```diff
@@ -23,6 +23,7 @@ dependencies:
   dynamic_color: ^1.7.0
   dynamic_themes: ^1.1.0
   expandable: ^5.0.1
+  file_picker: ^8.0.5
```

Comment:
to use the latest version we need to ubump the minimum Android SDK version to 21

Reply:
what is it right now/what android version does it correspond to

Reply:
not sure, but i tried bumping it to the latest but in the next version they dropped support and the build fails

https://pub.dev/packages/file_picker/changelog


I'll take a deeper look soon enough

Reply:
@oSumAtrIX appears to be Android SDK 19 (A4.4)